### PR TITLE
Add piqilib-0.6.15

### PR DESCRIPTION
**Pull request by alavrik**

### CODE REVIEW

**Comment by alavrik** [comment]
should be good to go. Can you merge this?

the test failure on osx is due to missing  CXXFLAGS='-std=c++14' and only triggered when protobuf is installed. I'll disable running protobuf-related tests by default in future versions so that opam doesn't stumble on problems like this

File: packages/piqilib/piqilib.0.6.15/opam [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  ["./configure" "--prefix" prefix]
+  [make]
+  [make "ocaml"]
+  [make "test"] {with-test}
+]
+install: [
+  [make "install"]
+  [make "ocaml-install"]
+]
+remove: [
```
Comment (mseri) [inline packages/piqilib/piqilib.0.6.15/opam]: This and the relative `flags` Are no longer needed since opam2

Comment (alavrik) [reply packages/piqilib/piqilib.0.6.15/opam]: @mseri can you clarify? Do you mean the whole remove section is no longer needed? How does opam know what to remove without it?

For the flags, do you mean the `flags: light-uninstall` line?

Comment (mseri) [reply packages/piqilib/piqilib.0.6.15/opam]: Yes, you can remove the whole remove and flags sections. Opam2 tracks all installed packages in its folder. In the end, you need to keep the remove and flags sections only if you are making changes to files in other packages that need to be reverted.

Comment (alavrik) [reply packages/piqilib/piqilib.0.6.15/opam]: @mseri this is done now

**Comment by mseri** [comment]
Before I merge can you check if the failure on Travis osx test is expected?

**Comment by alavrik** [comment]
@mseri yes, the test failure is expected. Ok to merge with it. See my first comment in this pull request for details.

**Pull request merged**